### PR TITLE
Dotnet 8 GA availability

### PR DESCRIPTION
**Pull request by tulikac**

### CODE REVIEW

**Comment by jeffwmartinez** [comment]
@tulikac this just passed last week.  Service principal is new so it might be a bug.  I'll try updating the action to get this to pass.

**Comment by jeffwmartinez** [comment]
> @tulikac this just passed last week. Service principal is new so it might be a bug. I'll try updating the action to get this to pass.

Updating the action version worked.  Please re-submit the blogs in a separate PR.

File: _posts/2023-11-14-dotnet8_ga.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: ".NET 8 GA available on App Service"
+author_name: "Tulika Chaudharie"
+---
+
+We have completed the rollout for [.NET 8 GA](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) support on App Service.
+
+Like in previous years we are using the [App Service Early Access feature](https://aka.ms/app-service-early-access) to enable day-0 support on the platform across all public regions on Windows App Service Plans. The early access release will be followed by additional deployments to fully integrate the new bits across our fleet, expecting to be fully done by the end of the week.
+
+On Linux App Service Plans, .Net 8 GA will be fully integrated on day-0 across all public regions without any additional deployments.
```
Comment (Mehul) [inline _posts/2023-11-14-dotnet8_ga.md:10]:
```suggestion
On Linux App Service Plans, .NET 8 GA will be fully integrated on day-0 across all public regions without any additional deployments.
```

Comment (Mehul) [reply _posts/2023-11-14-dotnet8_ga.md:10]: .NET should always be capitalized, thanks.

**Pull request closed without merging**